BlitzBoard/Features/SettingsScreenViewController.swift

```diff
@@ -3,7 +3,15 @@
 //  BlitzBoard
 //
 //  Created by Kiasha Rangasamy on 2024/05/06.
-//
 
+// #warning Settings File will be used in the future
```

Comment:
#warning Settings File will be used in the future
Don't put a warning in a comment. that defeats the purpose.
Please research how to use warnings in iOS